packages/platform-android/src/commands/runAndroid/getAdbPath.ts

```diff
@@ -8,7 +8,7 @@
 
 function getAdbPath() {
   return process.env.ANDROID_HOME
-    ? `${process.env.ANDROID_HOME}/platform-tools/adb`
+    ? `${process.env.ANDROID_HOME}/platform-tools/adb.exe`
```

Comment:
We're missing a check for `process.platform === 'win32'` here, as not everyone is on Windows :) Can you update it so this change only affects Windows users? Also cc @molant

Reply:
I already fix mine by changing the line ending. So it finally works without .exe. And I think this is actually specific in my environment

Reply:
Let's close this then